explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -67,13 +73,30 @@ import lucuma.ui.table.*
 import lucuma.ui.table.ColumnSize.*
 import lucuma.ui.table.hooks.*
 import lucuma.ui.utils.*
+import spire.math.*
 
 import java.text.DecimalFormat
 import scala.collection.decorators.*
 import scala.concurrent.duration.*
+import scala.language.implicitConversions
 
 import scalajs.js
 
+// TODO: These constants should likely go to lucuma-core
+type ArcSecondPerPixel = ArcSecond / Pixels
+type PixelScale        = Quantity[BigDecimal, ArcSecondPerPixel]
+
+val GmosPixelScale: PixelScale = BigDecimal(0.0807).withUnit[ArcSecondPerPixel]
+val F2PixelScale: PixelScale   = BigDecimal(0.18).withUnit[ArcSecondPerPixel]
```

Comment:
This looks like it should be part of the `Instrument` enum. Or maybe we can put the next 2 methods in extensions for each instrument? It seems the constants are not needed elsewhere.

Reply:
Yes, I'll move them to core